Lib/idlelib/configdialog.py

```diff
@@ -210,7 +210,7 @@ def help(self):
         """
         page = self.note.tab(self.note.select(), option='text').strip()
         view_text(self, title='Help for IDLE preferences',
-                 text=help_common+help_pages.get(page, ''))
+                  contents=help_common + help_pages.get(page, ''))
```

Comment:
The style in IDLE for arugments is generally to not have spaces around the `+`, so no need to change this.